modules/templates/helper.go

```diff
@@ -555,7 +555,8 @@ func SVG(icon string, others ...interface{}) template.HTML {
 			svgStr = heightRe.ReplaceAllString(svgStr, fmt.Sprintf(`height="%d"`, size))
 		}
 		if class != "" {
-			svgStr = strings.Replace(svgStr, `class="`, fmt.Sprintf(`class="%s `, class), 1)
+			matches := regexp.MustCompile(`class="(.*?)"`).FindStringSubmatch(svgStr)
```

Comment:
This is personal taste: I don't want to have `<svg>` elements generated by `svg.js` and `helper.go` to have different order of class names. I originally planned to deduplicate the class names (like in `svg.js`) but that would add unnecessary costs with no added benefit.

Reply:
You should move `regexp.MustCompile(`class="(.*?)"`)` out of the function to not compile the regex every function call.